src/jdk.crypto.mscapi/windows/native/libsunmscapi/security.cpp

```diff
@@ -943,7 +945,6 @@ JNIEXPORT jbyteArray JNICALL Java_sun_security_mscapi_CSignature_signCngHash
             dwFlags = 0;
             break;
         case 1:
-            BCRYPT_PKCS1_PADDING_INFO pkcs1Info;
```

Comment:
Don't lines 1181 and 1196 have the same problem?

Reply:
Good catch! Interestingly, that code appears to compile correctly. I'll fix it anyway.